testsuite/pytests/test_eprop_plasticity.py

```diff
@@ -208,40 +208,36 @@ def test_eprop_regression():
     weights_out_rec = np.array(np.random.randn(n_rec, n_out) / np.sqrt(n_rec), dtype=dtype_weights)
 
     params_common_syn_eprop = {
-        "batch_size": n_batch,
-        "optimizer": "gradient_descent",
+        "optimizer": {
+            "type": "gradient_descent",
+            "batch_size": n_batch,
+            "eta": 1e-4,
+            "Wmin": -100.0,
+            "Wmax": 100.0,
```

Comment:
```suggestion
            "batch_size": n_batch,
            "eta": 1e-4,
            "Wmax": 100.0,
            "Wmin": -100.0,
```
The suggested change sorts the parameters after the `type` alphabetically.

Reply:
Not accepting this since we want Wmin before Wmax. For the others above I will fix that on next push.